contracts/ocr2/src/contract.rs

```diff
@@ -1196,18 +1196,21 @@ pub fn query_oracle_observation_count(deps: Deps, transmitter: String) -> StdRes
 
 // Returns amount in juels
 fn calculate_reimbursement(
-    recommended_gas_price: u128,
+    config: &Billing,
     juels_per_luna: u128,
     signature_count: usize,
 ) -> Uint128 {
-    const BASE_GAS: Decimal = decimal(338_000);
-    const GAS_PER_SIGNATURE: Decimal = decimal(67_000);
+    let signature_count = decimal(signature_count as u64);
+    let gas_per_signature = decimal(config.gas_per_signature.unwrap_or(17_000));
+    let base_gas = decimal(config.base_gas.unwrap_or(84_000));
+    let gas_adjustment = Decimal::percent(u64::from(config.gas_adjustment.unwrap_or(140)));
```

Comment:
Adjusted based on devnet values. We seemed to over-estimate by almost exactly 4x. The amount charged is also the "gas allocated", not "gas used" and the difference seems to be about 1.4x